src/main/webapp/EditProfilePage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body>
+	<h1>Edit Your Profile Page</h1>
+	<form action="/profile-data" method="POST">
+		<label for="name">Name:</label><br>
+		<input type="text" id="name" name="name"><br>
+		<label for="email">Email:</label><br>
+		<input type="email" id="email" name="email"><br>
```

Comment:
You should probably add a 'required' tag for the input elements that you consider to be essential.
https://www.w3schools.com/tags/att_input_required.asp